### PR TITLE
Upgrade JMH to 1.20

### PR DESCRIPTION
**Pull request by mbasmanova**
#136

### CODE REVIEW

**Comment by martint** [comment]
Please update the CHANGES file

**Comment by mbasmanova** [comment]
Updated CHANGES file.

**Comment by electrum** [comment]
Please also check if Airlift and Presto need changes when updating to this version of JMH. This was required the last time I updated JMH. If so, put up pull requests for those that temporarily set the version using the property.

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+Airbase 80
+
+* Upgrade JMH to 1.20
```
Comment (electrum) [inline CHANGES.md]: Use the standard "Dependency updates" format. See below for examples.

**Comment by mbasmanova** [comment]
Updated CHANGES.md to use "Dependency updates" format.

@electrum , I set JMH version to 1.20 in dependency management section of Presto's parent POM file, ran `mvn install -DskipTests` and ran BenchmarkGeometryToBingTiles. I made the same changes in Airlift, ran `mvn install -DskipTests` and BenchmarkQuantileDigest. All succeeded. Are there any other checks that you think are necessary?

```
<dependencyManagement>
        <dependencies>
            <dependency>
                <groupId>org.openjdk.jmh</groupId>
                <artifactId>jmh-core</artifactId>
                <version>1.20</version>
            </dependency>

            <dependency>
                <groupId>org.openjdk.jmh</groupId>
                <artifactId>jmh-generator-annprocess</artifactId>
                <version>1.20</version>
            </dependency>
```

**Pull request merged**